app/poros/item.rb

```diff
@@ -0,0 +1,14 @@
+class Item 
+  attr_reader :item_name,
+              :item_id,
+              :item_description,
+              :item_price
+
+  def initialize(item)
+    @item_name = item[:attributes][:name]
+    @item_id = item[:id]
```

Comment:
good call at naming this attribute something other than "id" since "id" is a default attribute of all ruby objects.